gateway/conf/nginx.conf.liquid

```diff
@@ -106,6 +106,24 @@ http {
     {% include "conf.d/apicast.conf" %}
   }
 
+  {% if port.metrics %}
+    lua_shared_dict prometheus_metrics 16M;
```

Comment:
Should this be configurable?

Reply:
Probably, but I'd wait with some real operations use to see if it is really needed.

Reply:
Hah. This just gave me an idea for another exporter: shmem dict stats about free space.

Reply:
Done in https://github.com/3scale/apicast-cloud-hosted/pull/5/commits/09f65c93775c4bf6186aac8c92cdd06344e4f63d